src/App.jsx

```diff
@@ -1,40 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTabId, setSelectedId] = useState(tabs[0].id);
+  const [selectedTitle, setSelectedTitle] = useState(tabs[0].title);
```

Comment:
It's a good idea, but maybe it would be better to simply compare the array elements by id, which comes from the child element, so as not to create a new state.
But I'm not sure

Reply:
@ihorid Yes, it is worth doing as described in the comment above.